From 16812f93339fddb56377adacca74224f7eac3d48 Mon Sep 17 00:00:00 2001
From: me-cafebabe <me_cafebabe@hotmail.com>
Date: Mon, 2 May 2022 04:08:45 +0800
Subject: [PATCH] prima: Import xiaomi pine wlan mac changes

---
 CORE/VOSS/src/vos_nvitem.c | 34 ++++++++++++++++++++++++++++++++++
 1 file changed, 34 insertions(+)

diff --git a/CORE/VOSS/src/vos_nvitem.c b/CORE/VOSS/src/vos_nvitem.c
index d083f6acdb..d94fb0bd7f 100644
--- a/CORE/VOSS/src/vos_nvitem.c
+++ b/CORE/VOSS/src/vos_nvitem.c
@@ -810,6 +810,30 @@ VOS_STATUS vos_nv_get_dictionary_data(void)
 fail:
    return vosStatus;
 }
+//added by liulekang start
+VOS_STATUS get_mac_buf(char *pfileName,v_VOID_t *pCtx, char* mac, int len)
+{
+    int status;
+    VOS_STATUS retval = VOS_STATUS_SUCCESS;
+    hdd_context_t *pHddCtx = (hdd_context_t*)pCtx;
+    status = request_firmware(&pHddCtx->fw, pfileName, pHddCtx->parent_dev);
+    if(status || !pHddCtx->fw || !pHddCtx->fw->data) {
+        hddLog(VOS_TRACE_LEVEL_FATAL, "%s: Firmware %s download failed",
+                __func__, pfileName);
+        retval = VOS_STATUS_E_FAILURE;
+    }
+    else {
+        memcpy(mac, (v_VOID_t *)pHddCtx->fw->data, len);
+        hddLog(VOS_TRACE_LEVEL_FATAL, "%s: Firmware size",
+                        __func__);
+        /*hddLog(VOS_TRACE_LEVEL_FATAL, "%s: Firmware size = %d",
+                __func__, pHddCtx->fw->size);*/
+
+        retval = VOS_STATUS_SUCCESS;
+    }
+    return retval;
+}
+//added by liulekang end
 
 /**------------------------------------------------------------------------
   \brief vos_nv_parseV2bin() - Parse NV2 binary
@@ -1160,6 +1184,7 @@ VOS_STATUS vos_nv_open(void)
     v_SIZE_t nvReadBufSize;
     v_BOOL_t itemIsValid = VOS_FALSE;
     v_U32_t dataOffset;
+    char mac_buf[24];
     sHalNv *pnvData = NULL;
     hdd_context_t *pHddCtx = NULL;
 
@@ -1374,6 +1399,15 @@ VOS_STATUS vos_nv_open(void)
         /* NV verion is NV2 */
         ((VosContextType*)(pVosContext))->nvVersion = E_NV_V2;
     }
+    status = get_mac_buf("wlan/prima/wlan_mac.bin", ((VosContextType*)(pVosContext))->pHDDContext, mac_buf, sizeof(mac_buf));
+    //added return status
+    if (VOS_IS_STATUS_SUCCESS( status ))
+    {
+        memcpy(gnvEFSTable->halnv.fields.macAddr, mac_buf, VOS_MAC_ADDRESS_LEN);
+        memcpy(gnvEFSTable->halnv.fields.macAddr2, mac_buf + VOS_MAC_ADDRESS_LEN, VOS_MAC_ADDRESS_LEN);
+        memcpy(gnvEFSTable->halnv.fields.macAddr3, mac_buf + VOS_MAC_ADDRESS_LEN * 2, VOS_MAC_ADDRESS_LEN);
+        memcpy(gnvEFSTable->halnv.fields.macAddr4, mac_buf + VOS_MAC_ADDRESS_LEN * 3, VOS_MAC_ADDRESS_LEN);
+    }
 
     if (NULL != pnvData)
     {
-- 
2.30.2

